src/test/java/Tests/UpdateProfileTests.java

```diff
@@ -0,0 +1,212 @@
+package Tests;
+import java.text.SimpleDateFormat;
+
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import Utils.DriverSetup;
+import pages.HomePage;
+import pages.LoginPage;
+import pages.UserProfilePage;
+
+public class UpdateProfileTests extends DriverSetup{
+
+	LoginPage loginPage ;
+	HomePage homePage;
+	UserProfilePage userProfilePage;
+	
+	@BeforeClass
+	public void driversetup() {
+		loginPage = new LoginPage(driver);
+		homePage = new HomePage(driver);
+        userProfilePage = new UserProfilePage(driver);
+        
+        }
+	
+	@Test(priority=0, description = "Verifying Updating Profile Pic  Cancel Button")
+	public void VerifyUpdatingProfilePicCancelButton() throws Exception{
+		loginPage.login("addy3@mailinator.com", "password");
+		Thread.sleep(5000);
```

Comment:
Use explicit wait at all places. Also use println in all the tests at relevant places to tell of what is happening. It would be helpful in debugging